src/main/java/ch/njol/skript/expressions/ExprVelocity.java

```diff
@@ -62,38 +63,38 @@ protected String getPropertyName() {
 	@Override
 	@SuppressWarnings("null")
 	public Class<?>[] acceptChange(final Changer.ChangeMode mode) {
-		if ((mode == Changer.ChangeMode.SET || mode == Changer.ChangeMode.ADD || mode == Changer.ChangeMode.REMOVE || mode == Changer.ChangeMode.DELETE))
+		if (!(mode == Changer.ChangeMode.REMOVE_ALL))
 			return new Class[] {Vector.class};
 		return null;
 	}
 	
 	@Override
 	@Nullable
-	public Vector convert(Entity e) {
-		return e.getVelocity();
+	public Vector convert(Entity entity) {
+		return entity.getVelocity();
 	}
 	
 	@Override
 	@SuppressWarnings("null")
-	public void change(final Event e, final @Nullable Object[] delta, final Changer.ChangeMode mode) throws UnsupportedOperationException {
-		for (final Entity ent : getExpr().getArray(e)) {
-			if (ent == null)
-				return;
+	public void change(final Event event, final @Nullable Object[] delta, final Changer.ChangeMode mode) throws UnsupportedOperationException {
+		if (delta == null) return;
+		for (final Entity entity : getExpr().getArray(event)) {
+			if (entity == null) return;

```

Comment:
Wouldn't it make more sense to `continue;` (skipping over null elements) instead?

Reply:
That's the way Sashie had it, I would add a null check for the expression with the delta null check.

Reply:
don't throw me under the bus, it was bi0s code to begin with...